### PR TITLE
chore(): update samples.md page to use SDS instead of Qi

### PR DESCRIPTION
**Pull request by scotthowl**

### CODE REVIEW

File: Documentation/samples.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ uid: samples
 Samples
 =======
 
-The Qi-Samples illustrate several ways for applications to interact with the Qi REST API.
+The SDS-Samples illustrate several ways for applications to interact with the SDS REST API.
```
Comment (mckenzielee) [inline Documentation/samples.md]: This should be OCS-Samples

File: Documentation/samples.md [hunk 2]
```diff
@@ -22,5 +22,5 @@ Currently, the samples are available in these languages:
 * Python
 
 Because the examples are intended for demonstration purposes, they represent some example
-practices. The patterns may change as the Qi Preview progresses. Be sure
-to follow the Qi-Samples repository on GitHub for updates.
+practices. The patterns may change as the SDS Preview progresses. Be sure
+to follow the SDS-Samples repository on GitHub for updates.
```
Comment (mckenzielee) [inline Documentation/samples.md]: OCS-Samples

File: Documentation/samples.md [hunk 3]
```diff
@@ -22,5 +22,5 @@ Currently, the samples are available in these languages:
 * Python
 
 Because the examples are intended for demonstration purposes, they represent some example
-practices. The patterns may change as the Qi Preview progresses. Be sure
-to follow the Qi-Samples repository on GitHub for updates.
+practices. The patterns may change as the SDS Preview progresses. Be sure
```
Comment (mckenzielee) [inline Documentation/samples.md]: OCS Preview

File: Documentation/samples.md [hunk 4]
```diff
@@ -5,15 +5,15 @@ uid: samples
 Samples
 =======
 
-The Qi-Samples illustrate several ways for applications to interact with the Qi REST API.
+The OCS-Samples illustrate several ways for applications to interact with the SDS REST API.
```
Comment (osimarkrogers) [inline Documentation/samples.md]: "OCS-Samples" or "SDS-Samples"?

Comment (scotthowl) [reply Documentation/samples.md]: According to a previous comment on this PR, OCS-Samples is what we should use.

**Comment by osimarkrogers** [comment]
Merged.

**Pull request merged**